### PR TITLE
Add some new stuff

### PR DESCRIPTION
**Pull request by nayrpg**
# He can't

### CODE REVIEW

**Comment by nayrpg** [comment]
HE KHANT

**Comment by nayrpg** [comment]
👍 👍

**Comment by nayrpg** [comment]
💯 🔥 👌

**Comment by nayrpg** [comment]
😂 👌

**Comment by nayrpg** [comment]
(╯°□°）╯︵ ┻━┻

File: Dockerfile [hunk 1]
```diff
@@ -1,12 +1,9 @@
 FROM python:3.4.8-jessie
-
 WORKDIR /usr/src/app
+EXPOSE 8080
```
Comment (rygrange) [inline Dockerfile:3]: You are leaving yourself exposed! Cover your butt

File: game_session.py [hunk 2]
```diff
@@ -1,57 +0,0 @@
...
-    sql = '''insert into GAME_SESSION (start_time,state) values (datetime('now'), 'WAITING')'''
-    cur = conn.cursor()
-    cur.execute(sql)
-    session_id = cur.lastrowid
-    return session_id
-
-def join_session(conn, user_id, session_id):
-    sql = 'update GAME_SESSION set player_1=\''+ str(user_id) + '\' where session_id =\'' + str(session_id) + '\''
-    cur = conn.cursor()
-    cur.execute(sql)
```
Comment (rygrange) [inline game_session.py:15]: Actually... We needed this file

File: lazors/backend/backend.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    """
+    Deletes a player
+
+    PARAMETERS:
+        player_id: player id
+    RETURNS:
+        player
+    """
+    logging.info("Deleting player {}".format(player_id))
+    return True
```
Comment (rygrange) [inline lazors/backend/backend.py:56]: Add the clean your pet API

File: lazors/etc/apispec/swagger.yaml [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+
+
+securityDefinitions:
+  oauth2:
+    type: oauth2
+    flow: implicit
+    authorizationUrl: https://example.com/oauth2/dialog
+    scopes:
+      uid: Unique identifier of the user accessing the service.
```
Comment (rygrange) [inline lazors/etc/apispec/swagger.yaml:103]: Unnecessary comment

**Comment by nayrpg** [comment]
https://www.youtube.com/watch?v=R5JjdFqaVaw

File: lazors/etc/tls/server.crt [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+KcTI2rZVugQrGRTQ677DG6r3kiyXeklWB/BP4Jou8PkbBQbUeLXqVTuH6FMw6pKh
+KVjlFVLeN0mFVcsjAdy7BG6vKFguprkeT6+5pV+PwEs8B6r6WJdQJSShFvE1Km4G
+H7E1VYyWdEErIAJPoTEArpgJCiDvUD02F5+8T171s2VpiLd+lqF/zpG2cqTYIimI
+lSKKOFusCBH7SXnbfdgYiKOCGCVsqFTS/5k5dAwJ0sKnQbwFRW+yGy+RAjtWLv9q
+rh7yvpSF4KNBkSf050B7KvCo0Q3zIsdxUExpzo9mEfsC0BEd6HEqUYsfzP+6SNxo
+0exr+veTLoKf77fsHZySe0gsxz6UHqVWhm3+WUMVwZhBvSDi1bA6aGmtgplQny/L
+UXo+N9vbcH9AfqZRE9mw+AOOnbjQg87xKw6+Z3mopKBrLU+NpDs4kACi8u/TTwM2
+CBVoG2VeAFBurAD9F+hna6Wz1qzCIoNsxnJu6QufARl5G0v7Zl6Q4Uc85A7cteMh
+f8KYpZW/mR7CKqP2EhMx
+-----END CERTIFICATE-----
```
Comment (rygrange) [inline lazors/etc/tls/server.crt:33]: Remove large comment file

File: lazors/lazors.py [hunk 6]
```diff
@@ -0,0 +1,75 @@
+#!/usr/local/bin/python3
```
Comment (rygrange) [inline lazors/lazors.py:1]: Use python64

File: requirements.txt [hunk 7]
```diff
@@ -1 +1,2 @@
 connexion==1.4
+gevent==1.2.2
```
Comment (rygrange) [inline requirements.txt:2]: This file is now bloated as hell

**Review by rygrange — CHANGES_REQUESTED** [review]
Sucks!! Fix it!

File: requirements.txt (hunk 7 above)
Comment (nayrpg) [reply requirements.txt:2]: I agree requirements are lame

File: lazors/lazors.py (hunk 6 above)
Comment (nayrpg) [reply lazors/lazors.py:1]: 👌

File: lazors/etc/tls/server.crt (hunk 5 above)
Comment (nayrpg) [reply lazors/etc/tls/server.crt:33]: 🌚

File: lazors/etc/apispec/swagger.yaml (hunk 4 above)
Comment (nayrpg) [reply lazors/etc/apispec/swagger.yaml:103]: 🦄

File: lazors/backend/backend.py (hunk 3 above)
Comment (nayrpg) [reply lazors/backend/backend.py:56]: 🐶 🐃 💧

File: game_session.py (hunk 2 above)
Comment (nayrpg) [reply game_session.py:15]: ¯\_(ツ)_/¯

Comment (nayrpg) [reply game_session.py:15]: ¯\\\_(ツ)_/¯

File: Dockerfile (hunk 1 above)
Comment (nayrpg) [reply Dockerfile:3]: 🚽

**Pull request merged**